<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginpage</class>
 <widget class="QDialog" name="loginpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>travelSphere</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/Vector-3.png</normaloff>../../../Downloads/Vector-3.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#loginpage{

	background-color: qlineargradient(spread:pad, x1:0.280788, y1:0.0113636, x2:0.330039, y2:1, stop:0 rgba(161, 228, 231, 255), stop:1 rgba(255, 255, 255, 255));
}
*{
font-family:&quot;Times New Roman&quot;;
}</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>791</width>
     <height>580</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(123, 193, 215, 1), stop:1 rgba(126, 174, 218, 75));
border-radius:50px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">label{
font-family:Futura;
color:balck;
font-size:15px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
color:red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="userName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>500</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border-radius:10px;
color:grey;
background:#F5E0E0;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passWord">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>500</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border-radius:10px;
color:grey;
background:#F5E0E0;

}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>229</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>30</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border-radius:9px;
	background-color: qlineargradient(spread:pad, x1:0.0788177, y1:0.380682, x2:1, y2:0, stop:0.0640394 rgba(250, 130, 78, 255), stop:1 rgba(224, 206, 107, 255));
color:white;
font-family:&quot;Times New Roman&quot;;
	border-color: rgb(0,0,0);
  border-style: solid;
  border-width: 1px;

}

*:hover{
color:#F5E0E0;
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="icon">
     <iconset theme="window-new">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="signUp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border-radius:12px;
background-color:#5B7279;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Don't have an account? Sign Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forgotPW">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border-radius:13px;
background-color:#5B7279;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Forgot Password</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>371</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_2{
background-image: url(:/images/images/pickAndGo.png) ; 
background-repeat: norepeat;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
